lib/camt_parser/053/statement.rb

```diff
@@ -39,7 +39,7 @@ def electronic_sequence_number
 
       def opening_balance
         @opening_balance ||= begin
-          bal = @xml_data.xpath('Bal/Tp//Cd[contains(text(), "PRCD")]').first.ancestors('Bal')
+          bal = @xml_data.xpath('Bal/Tp//Cd[contains(text(), "OPBD") or contains(text(), "PRCD")]').first.ancestors('Bal')
```

Comment:
So depending whether it is `OPBD` or `PRCD`, the balance date has a different meaning. But currently the balance type is not parsed. I think I would not use it. What do you think?